src/BlazorWebView/src/Microsoft.AspNetCore.Components.WebView.Maui/iOS/BlazorMauiWebViewHandler.iOS.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using Foundation;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Handlers;
+using Microsoft.Maui.Hosting;
+using UIKit;
+using WebKit;
+using RectangleF = CoreGraphics.CGRect;
+
+namespace Microsoft.AspNetCore.Components.WebView.Maui
+{
+	public partial class BlazorWebViewHandler : AbstractViewHandler<IBlazorWebView, WKWebView>
+	{
+		static WKProcessPool? SharedPool;
+
+		protected override WKWebView CreateNativeView()
+		{
+			return new WKWebView(RectangleF.Empty, CreateConfiguration())
+			{
+				BackgroundColor = UIColor.Clear,
+				AutosizesSubviews = true
+			};
+		}
+
+		// https://developer.apple.com/forums/thread/99674
+		// WKWebView and making sure cookies synchronize is really quirky
+		// The main workaround I've found for ensuring that cookies synchronize 
+		// is to share the Process Pool between all WkWebView instances.
+		// It also has to be shared at the point you call init
+		static WKWebViewConfiguration CreateConfiguration()
+		{
+			var config = new WKWebViewConfiguration();
+
+			if (SharedPool == null)
+				SharedPool = config.ProcessPool;
+			else
+				config.ProcessPool = SharedPool;
+
+			return config;
+		}
+
+		public override Size GetDesiredSize(double widthConstraint, double heightConstraint)
+		{
+			SetDesiredSize(widthConstraint, heightConstraint);
+
+			return base.GetDesiredSize(widthConstraint, heightConstraint);
+		}
+
+		public void LoadHtml(string? html, string? baseUrl)
```

Comment:
Anything public I'd say put in an extension method like we do here
https://github.com/dotnet/maui/tree/main/src/Core/src/Platform/iOS

Also, these feel generally useful to Maui. If these seem like standard things we'll want to do against a WebView I'd say just create extension methods inside the Maui Core `WebViewExtensions` and just put this code there

If you think this code is only useful to BlazorWebView then leaving it here is probably fine

Reply:
I think this method will go away entirely (I don't think it's used; a legacy of its regular WebView origins).